Code/client/Games/Skyrim/Actor.cpp

```diff
@@ -352,17 +355,15 @@ void Actor::StopCombat() noexcept
     s_pStopCombat(this);
 }
 
-void Actor::RemoveSpell(MagicItem* apSpell) noexcept
+bool Actor::RemoveSpell(MagicItem* apSpell) noexcept
 {
-    TP_THIS_FUNCTION(TRemoveSpell, void, Actor, MagicItem*);
-    POINTER_SKYRIMSE(TRemoveSpell, removeSpell, 38717);
     if (!apSpell)
     {
-        spdlog::error("Actor::RemoveSpell: apSpell is null");
-        return;
+        spdlog::error(__FUNCTION__ ": apSpell is null");
+        return FALSE;
```

Comment:
tiny detail, but C++ has built in bool types, stylistically I'd prefer those over the msvc typedef'd TRUE and FALSE.

Reply:
OK, I'll clean that up.

Also, found a bit more to delete in MagicTarget.cpp as cleanup.